packages/twenty-front/src/modules/object-record/hooks/useRecordChipData.ts

```diff
@@ -20,19 +22,26 @@ export const useRecordChipData = ({
     PreComputedChipGeneratorsContext,
   );
 
+  const { fieldDefinition } = useContext(FieldContext);
+  const updatedRecord = { ...record };
```

Comment:
let's avoid creating a fake record having a "name", this will create confusion in the code. Instead, we can passe to identifierChipGenerator and generateDefaultRecordChipData what we need